ports/esp32s2/boards/unexpectedmaker_feathers2_neo/mpconfigboard.mk

```diff
@@ -0,0 +1,20 @@
+USB_VID = 0x239A
+USB_PID = 0x80B5
```

Comment:
Why are you using this PID? I see it allocated to something else internally. Maybe request one from Espressif instead?

Reply:
Looks like you did get the PID from Espressif. The PID is correct but the VID is wrong. It should be `0x303a` for Espressif.

Reply:
I'll fork this and fix.

Reply:
Super sorry! I totally forgot I got the FS2 PID from Adafruit, so only updated the PID, not the VID.
Thanks for taking care of this @tannewt !